### DISCUSSION TITLE
OnlyKey Support

### DISCUSSION THREAD

**Discussion by trymeouteh** [Kunzisoft/KeePassDX#1740] [Ideas]
Please consider adding onlykey support. Onlykey is the only open source hardware key out there.

https://onlykey.io/

**Comment by Arbel-arad** [comment]
i tried using my onlykeys yesterday, they work fine (same as yubikeys)

**Reply by Opening-Button-8988** [reply]
I was having issues with this. I discovered (in addition to realising I need the Key Driver app [only available on Play store] and not the ykDroid app), I needed to enter Config Mode on my onlykey and enable HMAC User Input Mode (click Yes) in the GUI app under Preferences).
